src/ui/units/collections/components/CollectionActions/CollectionActions.tsx

```diff
@@ -135,29 +135,16 @@ export const CollectionActions = React.memo<Props>(
 
         return (
             <div className={b(null, className)}>
-                {collectionData && (
-                    <React.Fragment>
-                        {collectionsAccessEnabled && collectionData.permissions.listAccessBindings && (
-                            <Button className={b('access')} onClick={onEditAccessClick}>
-                                {i18n('action_access')}
-                            </Button>
-                        )}
-                        {collectionData.permissions.move && (
-                            <Button className={b('move')} onClick={onMoveClick}>
-                                {i18n('action_move')}
-                            </Button>
-                        )}
-                        {collectionData.permissions.update && (
-                            <Button className={b('edit')} onClick={onEditClick}>
-                                {i18n('action_edit')}
-                            </Button>
-                        )}
-                    </React.Fragment>
+                {collectionData && collectionData.permissions.move && (
```

Comment:
Let's place the "Move" button in the same place as on the workbook page, in the `...` button.

<img width="403" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/83e85914-43d6-4658-8e1f-118736299c78">

Reply:
if
<img width="896" alt="Screenshot 2023-10-04 at 10 23 02" src="https://github.com/datalens-tech/datalens-ui/assets/32207888/97ab7832-8173-42ad-a0a1-964d55851eb7">
 i will place button "Move", in button ... will be only 1 item

Reply:
added new btn move, near btn ...